frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailHeader.tsx

```diff
@@ -73,7 +73,6 @@ export function ObjectDetailHeader({
 
           {actionItems.length > 0 && (
             <EntityMenu
-              horizontalAttachments={["right", "left"]}
```

Comment:
this was the only usage of the horizontalAttachments prop, and it didn't seem to do much here, so I removed it.